### PR TITLE
Replace 0 and NULL with nullptr

### PR DESCRIPTION
**Pull request by Broxzier**

### CODE REVIEW

File: src/openrct2/drawing/TTFSDLPort.cpp [hunk 1]
```diff
@@ -357,13 +357,13 @@ static TTF_Font* TTF_OpenFontIndexRW(FILE* src, int freesrc, int ptsize, long in
         {
             fclose(src);
         }
-        return NULL;
+        return nullptr;
```
Comment (Gymnasiast) [inline src/openrct2/drawing/TTFSDLPort.cpp]: I deliberately left out this file because it's directly ~~stolen~~taken from SDL2-TTF.

Comment (Broxzier) [reply src/openrct2/drawing/TTFSDLPort.cpp]: Then wouldn't this fit better with the other dependencies, or inside the thirdparty folder?

File: src/openrct2/ride/RideRatings.cpp [hunk 2]
```diff
@@ -4440,16 +4440,16 @@ static const ride_ratings_calculation ride_ratings_calculate_func_table[RIDE_TYP
     ride_ratings_calculate_magic_carpet,                   // MAGIC_CARPET
     ride_ratings_calculate_submarine_ride,                 // SUBMARINE_RIDE
     ride_ratings_calculate_river_rafts,                    // RIVER_RAFTS
-    NULL,                                                  // 50
+    nullptr,                                                  // 50
```
Comment (janisozaur) [inline src/openrct2/ride/RideRatings.cpp]: the comments here are now misaligned

Comment (Broxzier) [reply src/openrct2/ride/RideRatings.cpp]: The new run-clang-format job did mark this too!

**Pull request merged**